### ISSUE TITLE
Uncaught TypeError: this.service.deduceType is not a function

### ISSUE THREAD

**Issue by Timemashine** [Gert-dev/php-integrator-call-tips#2]
[Enter steps to reproduce below:]

often appears when deleting brackets:

**Atom Version**: 1.6.2
**System**: Mac OS X 10.11.4
**Thrown From**: [php-integrator-call-tips](https://github.com/Gert-dev/php-integrator-call-tips) package, v0.2.3

### Stack Trace

Uncaught TypeError: this.service.deduceType is not a function

```
At /Users/timemachine/.atom/packages/php-integrator-call-tips/lib/Provider.coffee:48

TypeError: this.service.deduceType is not a function
    at Provider.module.exports.Provider.onChangeCursorPosition (/Users/timemachine/.atom/packages/php-integrator-call-tips/lib/Provider.coffee:48:22)
    at /Users/timemachine/.atom/packages/php-integrator-call-tips/lib/AbstractProvider.coffee:124:18
```

### Commands

```
     -0:51.4.0 core:move-left (atom-text-editor.editor.is-focused)
     -0:50.8.0 core:move-right (atom-text-editor.editor.is-focused)
  3x -0:49.9.0 core:move-left (atom-text-editor.editor.is-focused)
     -0:46.3.0 core:backspace (atom-text-editor.editor.is-focused)
  2x -0:45.6.0 core:move-right (atom-text-editor.editor.is-focused)
  2x -0:44.6.0 core:move-left (atom-text-editor.editor.is-focused)
  3x -0:43.2.0 core:move-right (atom-text-editor.editor.is-focused)
     -0:42.4.0 core:backspace (atom-text-editor.editor.is-focused)
  5x -0:41.6.0 core:move-down (atom-text-editor.editor.is-focused)
     -0:23.4.0 php-cs-fixer:fix (atom-text-editor.editor.is-focused)
  3x -0:15.2.0 click (div.close.icon.icon-x)
     -0:09.8.0 core:move-left (atom-text-editor.editor.is-focused)
  2x -0:09.5.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:05.3.0 core:move-right (atom-text-editor.editor.is-focused)
     -0:05.0 core:backspace (atom-text-editor.editor.is-focused)
  3x -0:04.4.0 core:move-down (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "gcc-make-run",
      "git-time-machine",
      "linter-shellcheck",
      "php-class-tree",
      "switch-header-source",
      "antialiased",
      "an-color-picker",
      "color-tabs",
      "epitools",
      "minimap",
      "project-sidebar",
      "project-environment",
      "drunken-php"
    ],
    "themes": [
      "native-ui",
      "poxy-monokai"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-beautify, v0.29.2
browser-plus, v0.0.57
file-icons, v1.7.1
font-viewer, v0.3.1
fonts, v1.2.0
hyperclick-php, v0.2.1
language-ini, v1.16.0
linter, v1.11.4
linter-php, v1.2.0
native-ui, v0.17.0
on-save, v0.2.9
pdf-view, v0.50.0
php-analyser, v0.2.0
php-composer-completion, v0.3.4
php-cs-fixer, v2.5.6
php-debug, v0.2.2
php-extract-method, v0.2.1
php-integrator-annotations, v0.4.0
php-integrator-autocomplete-plus, v0.7.0
php-integrator-base, v0.8.0
php-integrator-call-tips, v0.2.3
php-integrator-linter, v0.2.2
php-integrator-navigation, v0.6.2
php-integrator-refactoring, v0.4.2
php-integrator-symbol-viewer, v0.5.6
php-integrator-tooltips, v0.6.0
pigments, v0.26.0
poxy-monokai, v0.1.1
project-manager, v2.9.7
project-plus, v0.8.0
terminal-plus, v0.14.5
webbox-color, v0.5.9

# Dev
No dev packages
```

**Comment by Gert-dev** [comment]
This should be fixed in 0.2.4. Let me know if it pops up again.

**Comment by Timemashine** [comment]
everything works fine now. Thanks a lot for these Packages!
